core/src/main/java/cz/o2/proxima/transform/RenameTransformation.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2017-2020 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.transform;
+
+import cz.o2.proxima.annotations.Evolving;
+import cz.o2.proxima.functional.BiFunction;
+import cz.o2.proxima.functional.UnaryFunction;
+import cz.o2.proxima.repository.AttributeDescriptor;
+import cz.o2.proxima.repository.Repository;
+import cz.o2.proxima.storage.StreamElement;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+
+/** A stateless transformation that performs transformation from one attribute to another. */
```

Comment:
Can be this javadoc more detailed please? Ideally with usage example?

Reply:
Done.